.vale.ini

```diff
@@ -0,0 +1,7 @@
+StylesPath = .github/styles
+
+MinAlertLevel = suggestion
+Vocab = Base
+
+[*.md]
+BasedOnStyles = Vale, alex, proselint, Readability, write-good, dvc
```

Comment:
Most of these are available [packages](https://vale.sh/docs/topics/packages/), I think 🙂 that should mean there's no need to copy them into `styles/` in this repo, I think.

Reply:
> That should mean there's no need to copy them into styles/ in this repo (need to check).

 I didn't expect the packages to work for us out of the box and there is no way to modify the packages (i.e. add/edit/remove rules) unless we copy the contents.
 
 I think the best would be to create a single `dvc` package with rules cherry picked (copy-pasted / edited) from existing packages + our custom rules.